pkg/engine/lifecycletest/testdata/output/TestImportOption/0/diff.stdout.txt

```diff
@@ -1,5 +1,19 @@
 <{%fg 2%}>+ pulumi:providers:pkgA: (create)
 <{%fg 2%}>    [urn=urn:pulumi:test::test::pulumi:providers:pkgA::default]
-<{%reset%}><{%reset%}><{%fg 13%}><{%bold%}>Resources:<{%reset%}>
+<{%reset%}><{%reset%}><{%fg 2%}>= pkgA:m:typA: (import)
```

Comment:
The rest of the display changes here are just provider UUIDs changing (would be nice if they were deterministic for these tests 🤔) but this file is a real display change where now we show the typA resource being imported then updated.